lib/app_profiler/backend/stackprof.rb

```diff
@@ -0,0 +1,93 @@
+# frozen_string_literal: true
+
+require "stackprof"
+
+module AppProfiler
+  class StackprofBackend < Backend
```

Comment:
The naming here is wrong. Let's rename to `AppProfiler::Backend::Stackprof` to match the file name (also like how viewer and storage are defined).